main.py

```diff
@@ -112,7 +112,7 @@ def get_verdict():
 
         ideal_temperature = destination.Temperature
 
-        api_key = 'KaFIkTNjT3CgIbkFGYLY3633FiW71u67'
+        api_key = ''
```

Comment:
Almost. Need to reference the config param here.  So you would do something like:

import config
....
api_key = config.API_KEY

The blank string should be in config.py  next to a new config API_KEY and the reference in the main.py. this way when someone wants to change it, they only change values in config.py, not main